GOVERNANCE.md

```diff
@@ -181,24 +181,54 @@ yourself from the MAINTAINERS file.
 ## Removal of inactive maintainers
 
 Similar to the procedure for adding new maintainers, existing maintainers can
-be removed from the list if they do not show significant activity on the
-project. Periodically, the maintainers review the list of maintainers and their
-activity over the last year.
-
-If a maintainer has shown insufficient activity over this period, a neutral
-person will contact the maintainer to ask if they want to continue being
-a maintainer. If the maintainer decides to step down as a maintainer, they
-open a pull request to be removed from the MAINTAINERS file.
-
-If a committer is unable to continue participating in project level votes but
-wishes to remain a maintainer on the project, they may convert their role to
-a reviewer until they are able to perform committer duties again.
-
-If an inactive maintainer wants to remain in their current role or does not open
-their own pull request to change roles, another maintainer may open up a pull
-request to the MAINTAINERS file to change their role. Role changes and removals
-must be approved by 2/3 of the current committers. The voting and discussion
-period is seven days, after which a committer will verify the votes,
+be removed from the list if they do not show consistent activity on the
+project. Depending on the existing role, committer or reviewer, there are
+variations to the removal process detailed as follows.
+
+### Committer removal
+
+For committers, the maintainers periodically review committer project
+activity. The maintainers use the CNCF "devstats" community metrics tool for
+data on user activity statistics.
+
+If a committer has shown no activity over a six-month period, the committer
+will be contacted to notify them of their proposed Emeritus role. There is
+no automatic change of status in the project. First, the activity status is
+discussed directly with the committer, and second, maintainers discuss whether
+other non-tracked contributions to the project reflect an ongoing, active
+participation in the project.
+
+Based on the outcome of these discussions, the committer may choose to remove
+themselves from the MAINTAINERS file, or another maintainer may open a
+pull request to the MAINTAINERS file on their behalf for removal.
+Removals must be approved by 2/3 of the current committers. The voting
```

Comment:
```suggestion
Removals must be approved by 2/3 of the current committers who voted within 14 days. The voting
```

To avoid potential dead-lock when more than 1/3 of the committers are inactive.

Reply:
I don't think we need to change our threshold voting rules at this time. We have had some voting go right up to the 7 days, but even with 1/3 inactive right now, we have always gotten the votes. Even inactive committers are usually willing to come back on important votes, but the idea here is that in the future we will always have a comfortable 2/3 actively participating in voting an discussions.

We can continue to discuss evolving our governance model to better recognize both contributor activity as well as the wider diversity of containerd projects.